API/src/main/java/com/bgsoftware/superiorskyblock/api/config/SettingsManager.java

```diff
@@ -261,10 +255,28 @@ public interface SettingsManager {
     boolean isTeleportOnKick();
 
     /**
-     * Whether to clear players' inventories when they join a new island or not.
-     * Config-path: clear-on-join
+     * Whether to clear members Ender Chest when their island is disbanded or not.
+     * Config-path: clear-ender-chest-on-disband
+     */
+    boolean isClearEnderChestOnDisband();
+
+    /**
+     * Whether to clear players Ender Chest when they join a new island or not.
+     * Config-path: clear-ender-chest-on-join
+     */
+    boolean isClearEnderChestOnJoin();
+
+    /**
+     * Whether to clear players inventory when their island is disbanded or not.
+     * Config-path: clear-inventory-on-disband
+     */
+    boolean isClearInventoryOnDisband();
+
+    /**
+     * Whether to clear players inventory when they join a new island or not.
+     * Config-path: clear-inventory-on-join
      */
-    boolean isClearOnJoin();
```

Comment:
Do not delete the method.
Instead mark it as deprecated, change the comment to reflect what it really returns (for example, return true if one of the options are true) and in the future it will be deleted